production/helm/loki/src/dashboards/loki-logs.json

```diff
@@ -237,7 +237,7 @@
             "steppedLine": false,
             "targets": [
                {
-                  "expr": "sum(rate(container_cpu_usage_seconds_total{cluster=\"$cluster\", namespace=\"$namespace\", pod=~\"$deployment.*\", pod=~\"$pod\", container=~\"$container\"}[5m]))",
+                  "expr": "sum(rate(container_cpu_usage_seconds_total{namespace=\"$namespace\", pod=~\"$deployment.*\", pod=~\"$pod\", container=~\"$container\"}[5m]))",
```

Comment:
@trevorwhitney you've explained this to me but I've forgot again. Why was the cluster label an issue and was it required?

Reply:
It is a metric label/annotation on the goroutine or go specific metric info, but for the last several chart releases our version of this dashboard has been mostly empty because the pod metrics for CPU/MEM/etc don't have `cluter` label/annotations